subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/NupokatiPlugin.kt

```diff
@@ -65,16 +67,20 @@ public class NupokatiPlugin : Plugin<Project> {
 
                         this.artifactoryUser.set(extension.artifactory.login)
                         this.artifactoryPassword.set(extension.artifactory.password)
-                        this.artifactoryUploadPath.set(
-                            cdBuildConfig.map {
-                                it.outputDescriptor.path.substringBeforeLast('/')
-                            }
-                        )
-                        this.buildVariant.set(variant.name)
-                        this.files.set(project.files(bundle))
+                        this.artifactoryUploadPath.set(cdBuildConfig.map {
+                            it.outputDescriptor.path.substringBeforeLast('/')
+                        })
+                        this.schemaVersion.set(cdBuildConfig.map { it.schemaVersion })
                         this.statsDConfig.set(project.statsdConfig)
                         this.buildOutput.set(project.layout.buildDirectory.file("nupokati/buildOutput.json"))
 
+                        @Suppress("DEPRECATION")
+                        this.files.set(project.files(bundle))
+                        @Suppress("DEPRECATION")
+                        this.buildConfiguration.set(
```

Comment:
Why we can't name `buildConfiguration` as `buildType`?

Reply:
I think we can, it's the universal name that will be used in contract between ios and android i believe.
It is not android-specific and we can use any build variant we want, but from RELEASE/CD domain standpoint it would be some abstract "release" or "staging" or "test" configuration